test/functional/basic/gesture-e2e-specs.js

```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
         await doPinch();
       });
     });
+    describe('special actions', function () {
+      it('should open the control center by swiping up at the bottom', async function () {
+        await driver.elementByAccessibilityId('ControlCenterView')
+          .should.eventually.be.rejectedWith(/An element could not be located/);
+
+        const window = await driver.elementByClassName("XCUIElementTypeWindow");
+        const size = await window.getSize();
+
+        let action = new wd.TouchAction(driver);
+        action.press({
+          x: size.width / 2,
+          y: size.height - 5,
+        }).wait(500)
+        .moveTo({
+          x: size.width / 2,
+          y: size.height / 2,
+        });
+        await action.perform();
+
+        // Control Center ought to be visible now
+        await driver.elementByAccessibilityId('ControlCenterView');
```

Comment:
should.be.fulfilled?

Reply:
There's no particular point. If it throws an error the test will fail for it. If it is never resolved the test will time out.